src/driver/latin_hypercube_random_test_point_generator.py

```diff
@@ -48,11 +50,19 @@ def __init__(
           dataset:  The DataSet for which test points are to be generated.
           rng:  A numpy Generator object that is used to seed the generation
             of random test points.
+          npoints_generator: a function from the number of publishers to the
```

Comment:
Could we change the comment to read, 'A function that returns the number of test points that should be generated.  The argument of the function is the number of publishers.'

Reply:
Sure. It does read better.